src/ui/events.js

```diff
@@ -770,6 +775,11 @@ export type MapEvent =
      */
     | 'touchcancel'
 
+    /** @section Movement
```

Comment:
`Movement` is a weird category. Can it be folded into a single `Querying`,  `Map State` or `Camera` category?

Reply:
`Camera` sounds good to me!